src/AstromFit.cc

```diff
@@ -963,14 +961,10 @@ unsigned AstromFit::Minimize(const std::string &whatToFit, const double nSigRejC
     Eigen::VectorXd grad(_nParTot);  grad.setZero();
 
     //Fill the triplets
-    clock_t tstart = clock();
```

Comment:
It was cluttering up the output very badly, I now have a python profiling option, and the slow parts of the code are not in the guts now. If we need detailed timing, we can re-do it later with proper logging, when we decide we need it.